GoogleForCreators/web-stories-wp
# Update tab rendering logic and interaction

## Context

As per #8196, this does two things:

* Change the keyboard interaction for tab views to use manual interaction
* Renders only the current tab panel for tabbed displays

## User-facing changes

Only in the keyboard interaction mode for tab views.

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Click a tab in the library tab list
2. Use arrow keys and/or home/end to move focus to another tab
3. Observe that only tab focus changes, not the current tab
4. With another tab than the current tab focused, click space/enter
5. Observe that the current tab changes

### UAT




## Checklist




---

Fixes #8196
